### PR TITLE
Add getBlockedContacts method

### PR DESCRIPTION
**Pull request by Gimenz**
Simple method to get blocked ids

```javascript
const blockedIds = await client.getBlockedIds();
//['62813xxx@c.us', '628xxx@c.us']
```
it will return empty array if no one contact is blocked

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -931,6 +931,18 @@ class Client extends EventEmitter {
...
+    /**
+     * Get all blocked ids by Host account
+     * @returns {Promise<Array>}
+     */
+    async getBlockedList() {
+        let chatIds = await this.pupPage.evaluate(() => {
+            return Object.keys(window.Store.Blocklist._index);
+        });
+
+        return chatIds
```
Comment (tuyuribr) [inline src/Client.js:944]: Missing semicolon

**Review by pedroslopez — COMMENTED** [review]
Nice! Thanks for contributing.

I would suggest being a bit clearer on what kind of ID you're returning, for example `getBlockedContactIds`.

Better yet, it may be more useful to return contacts themselves instead of just their IDs (`getBlockedContacts()`).

**Review by purpshell — APPROVED** [review]
fixed a bunch of stuff for you, hope you don't mind. It was simple changes, and I didn't it to be delayed. anyways, looks good to me.

**Comment by Gimenz** [comment]
> fixed a bunch of stuff for you, hope you don't mind. It was simple changes, and I didn't it to be delayed. anyways, looks good to me.

thank you, I'm not very careful

**Review by pedroslopez — APPROVED** [review]
Added some tests and fixed it up a bit, thanks for contributing!

**Pull request merged**